subprojects/packagefiles/tree-sitter/meson.build

```diff
@@ -0,0 +1,28 @@
+project('tree-sitter', 'c'
+  version: '0.15.7',
+  license: 'MIT',
+)
+
+tree_sitter = library('tree-sitter', [
+  'lib/src/lib.c',
+  ],
+  include_directories: [
+    include_directories('lib/src'),
+    include_directories('lib/include'),
+    include_directories('lib/utf8proc')
+  ],
+)
+
+tree_sitter_dep = declare_dependency(
+  link_with: tree_sitter,
+  include_directories: include_directories('lib/include'),
+)
+
+
+pkg = import('pkgconfig')
+pkg.generate(libraries : tree_sitter,
+             subdirs : 'lib/include',
```

Comment:
Subdirs are supposed to reflect the installed location, which is /usr/include -- there is no /usr/include/lib/include subdir, and anyway upstream uses `Cflags: -I${includedir}` so there is no subdir.